### ISSUE TITLE
JAWS does not announce the required state on textareas when the placeholder attribute is used

### ISSUE THREAD

**Issue by douga11y** [FreedomScientific/VFO-standards-support#553]
### Summary

> When tabbing to a `<textarea>` that uses both the HTML5 `required` attribute and a non-empty `placeholder` attribute, the required state information is not announced by JAWS. This only occurs when tabbing to the form field. If navigating to the field using the arrow keys the required state information is announced as expected.
> This appears to be similar to the bug reported in [issue 484](https://github.com/FreedomScientific/VFO-standards-support/issues/484), but is related to the placeholder attribute rather than an actual text value in the textarea.
> 
> 1. With JAWS enabled, navigate to the codepen provided below
> 2. Tab to the "Comments" textarea

### Expected result

> JAWS should announce the required state information for the textarea.

### Actual result

> JAWS does not announce the required state information.

### Example

> [CodePen demo](https://cdpn.io/dabrams888/debug/rNmGNBz/GnAnbXzRXgXA)

#### JAWS version and build number
JAWS Version 2021.2102.34

 
#### Operating System and version
Microsoft Windows 10 Pro Version 10.0.19042 Build 19042

#### Browser and version:
Google Chrome Version 91.0.4472.164 (Official Build) (64-bit)

**Comment by beltonTPG** [comment]
I have also just found that `aria-describedby` content is not announced when `placeholder` text is present.

**Comment by JAWS-test** [comment]
The problem also occurs in Firefox and Edge. Is a problem of JAWS, because the information is correctly transmitted via the Accessibility API and also output by NVDA.

**Comment by JAWS-test** [comment]
The problem with textarea with placeholder is not limited to required, but also occurs with many other attributes, e.g.:

* aria-required
* aria-roledescription
* aria-current
* aria-invalid
* aria-haspopup
* aria-errormessage
* aria-disabled

All these attributes are not output by JAWS.

Only two relevant ARIA attributes are output:
* aria-keyshortcuts
* aria-readonly

**Comment by StephenBeckettTPGi** [comment]
Behaviour confirmed.

Also note: placeholder text is not read out by JAWS - JAWS says that the field "Contains text", which is not true.

**Comment by StephenBeckettTPGi** [comment]
Closing #554 as it is included here.

**Comment by Chacomo** [comment]
We have a lot of forms which have a required textarea in our application. Currently this state is only announced by NVDA.

One additional hint:
If the textarea is set readonly, then the required information will be announced.
The label for a readonly textarea will also be shown in the braille viewer (at the moment it is only read).

Example:
```
<span id="test-1">Test 1</span>
<textarea tabindex="0" aria-labelledby="test-1" aria-readonly="true" aria-required="true"></textarea>
<span id="test-2">Test 2</span>
<textarea tabindex="0" aria-labelledby="test-2" aria-required="true"/></textarea>
```

Do you already know whether this will be fixed? We would really appreciate it.
